src/widget/chatformheader.cpp

```diff
@@ -121,7 +121,7 @@ ChatFormHeader::ChatFormHeader(QWidget* parent)
     nameLabel->setMinimumHeight(Style::getFont(Style::Medium).pixelSize());
     nameLabel->setEditable(true);
     nameLabel->setTextFormat(Qt::PlainText);
-    connect(nameLabel, &CroppingLabel::editFinished, this, &ChatFormHeader::onNameChanged);
+    connect(nameLabel, &CroppingLabel::editFinished, this, [this](const QString& name){emit nameChanged(name);});
```

Comment:
AFAIK you can connect signal to signal. Just use
`connect(nameLabel, &CroppingLabel::editFinished, this, nameChanged);`